ansys/dpf/core/dimensionality.py

```diff
@@ -41,17 +41,17 @@ def __str__(self):
     
     @staticmethod
     def scalar_dim():
-        """Dimensionnality instance corresponding to a scalar field
+        """Dimensionality instance corresponding to a scalar field
 
         Returns
         -------
-        dimensionnality : Dimensionnality
+        dimensionality : Dimensionality
```

Comment:
We should consider spelling out the class return types like:

```suggestion
        :class:`ansys.dpf.core.dimensionality.Dimensionality`
```

One of the things we've been doing within `pyaedt` is allowing sphinx to provide links between of return types and class documentation.  It's super helpful.  Also, there's no reason to spell out the parameter name of the return type.

Reply:
Hi @akaszynski, dpf core documentation already provides links between the return type and the considered class (working with this spelling syntax)